run_bot.sh

```
@@ -52,9 +52,21 @@ check_docker() {
-    print_success "Docker image built successfully"
+    print_info "Tagging as: ${full_tag}"
+    docker build -t "${full_tag}" .
+    print_success "Docker image built successfully: ${full_tag}"
+    
+    # Also tag as latest for convenience
+    docker tag "${full_tag}" "${image_name}:latest"
+    print_info "Also tagged as: ${image_name}:latest"
+    
+    # Update the image name for the rest of the script
+    export DOCKER_IMAGE_NAME="${full_tag}"
 }
```

Shellcheck (0.10.0)

[warning] build_image references arguments, but none are ever passed.

(SC2120)